### PR TITLE
Add in text sanitizer, per https://github.com/samvera/hyrax/pull/3225

### PR DESCRIPTION
**Pull request by lfarrell**
Adding fix from hyrax pull request, since this is a file we override

### CODE REVIEW

**Comment by lfarrell** [comment]
Also see pull request https://github.com/samvera/hyrax/pull/3231

**Comment by bbpennel** [comment]
This is merged into hyrax now at least, i'm not quite sure what to do about a bug fix from a version of hyrax we aren't on yet, but i guess it wouldn't do any harm

**Comment by lfarrell** [comment]
Yes, but we won't pick up the changes since we've overridden the file. The joys of basically forking a file :)

File: app/helpers/hyrax/citations_behaviors/formatters/chicago_formatter.rb [hunk 1]
```diff
@@ -14,13 +14,13 @@ def format(work)
...
+          text << " #{whitewash(pub_date)}." unless pub_date.nil?
 
           text << format_title(work.to_s)
           pub_info = setup_pub_info(work, false)
-          text << " #{pub_info}." if pub_info.present?
+          text << " #{whitewash(pub_info)}." if pub_info.present?
           # UNC customization. Add DOI
-          text << " #{work.doi[0]}" if !work.doi.nil? && work.doi.length > 0
+          text << " #{whitewash(work.doi[0])}" if !work.doi.nil? && work.doi.length > 0
           text.html_safe
```
Comment (smithjp) [inline app/helpers/hyrax/citations_behaviors/formatters/chicago_formatter.rb]: There are still instances of html_safe in each of the citation formatter files

Comment (lfarrell) [reply app/helpers/hyrax/citations_behaviors/formatters/chicago_formatter.rb]: They're that way in master too. I think the main issue was trusting user provided titles, at least as far as this request went.

Comment (lfarrell) [reply app/helpers/hyrax/citations_behaviors/formatters/chicago_formatter.rb]: This first three in this list should be fixed https://github.com/samvera/hyrax/issues/3187, by these changes and upgrading to 2.2.x
The fourth issue is still a problem https://github.com/samvera/hyrax/issues/3232

**Pull request closed without merging**